appservice/src/main/java/nu/yona/server/subscriptions/rest/UserController.java

```diff
@@ -423,6 +426,18 @@ private static void addConfirmMobileNumberLink(Resource<UserDTO> userResource, S
 			userResource.add(UserController.getConfirmMobileLink(userResource.getContent().getID()));
 		}
 
+		private void addWeeksActivityLink(UserResource userResource)
```

Comment:
Good to see the links here too.
Please apply the renames as I have proposed for the controller methods.